### PR TITLE
Load-multiplier to weight servers

### PR DESCRIPTION
**Pull request by paul1278**
With this changes a load-multiplier can be added to the servers. It will be multiplicated with the server-load.

Also added a small check for the parameters when adding servers

### CODE REVIEW

**Comment by defnull** [comment]
We tested this successfully and are running this patch in production.

**Comment by paul1278** [comment]
@defnull Glad to hear that!

File: lib/tasks/servers.rake [hunk 1]
```diff
@@ -10,16 +10,29 @@ task servers: :environment do
     puts("\tsecret: #{server.secret}")
     puts("\t#{server.enabled ? 'enabled' : 'disabled'}")
     puts("\tload: #{server.load.presence || 'unavailable'}")
+    puts("\tload multiplier: #{server.load_multiplier}")
```
Comment (farhatahmad) [inline lib/tasks/servers.rake:13]: Can this be changed to `server.load_multiplier.nil? ? 1.0 : server.load_multiplier.to_d`

Comment (paul1278) [reply lib/tasks/servers.rake]: Done on fd9dcf7

**Pull request merged**